src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -35,6 +35,11 @@ public class LanguageServer
     {
         private const char EOL = '\n';
 
+        /// <summary>
+        /// This const represents the dummy formula that is used to create an infrastructure needed to get the symbols for Nl2Fx operation.
+        /// </summary>
+        public const string Nl2FxDummyFormula = "\"Dummy Nl2Fx Check\"";
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

Does this need to be public?

Reply:
Made this interval and set it to GUID as we discussed offline